docx/text/tabstops.py

```diff
@@ -18,3 +18,16 @@ class TabStops(ElementProxy):
     using the `tab_stops` property of ParagraphFormat; it is not intended to
     be constructed directly.
     """
+
+    __slots__ = ()
+
+    def __len__(self):
+        tabs = self._get_tabs()
+        return len(tabs.tab_lst)
+
+    def _get_tabs(self):
+        """
+        Get or Create tabs list as needed.
+        """
+        tabs = self._element.get_or_add_tabs()
+        return tabs
```

Comment:
I see you are putting the the TabStop class in the same file as the TabStops class, where I had them in separate files.  Is this how you want it going forward, or will we separate these classes into different files at a later time?